plinth/modules/tor/templates/tor.html

```diff
@@ -53,7 +53,11 @@
       {% endif %}
     </p>
 
-    {% include "diagnostics_button.html" with module="tor" %}
+    {% if status.is_running %}
+      {% include "diagnostics_button.html" with module="tor" %}
```

Comment:
Why not pass another argument `app_is_running` to the `diagnostics_button.html` template during include and have the same template for disabled or enabled buttons?